buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -58,7 +58,7 @@ object Plugins {
         scope.kotlin("jvm")
 
     fun ksp(scope: PluginDependenciesSpec) =
-        scope.id("com.google.devtools.ksp").version("1.6.10-1.0.2")
+        scope.id("com.google.devtools.ksp").version("1.7.0-1.0.6")
```

Comment:
```suggestion
        scope.id("com.google.devtools.ksp").version("1.7.10-1.0.6")
```

Reply:
Yep. I got plenty of warnings for missing that one out :D 

Addressed.